src/grid/styles.scss

```diff
@@ -7,71 +7,67 @@
 
 @use '../internal/styles' as styles;
 @use '../internal/styles/tokens' as awsui;
+@use '../internal/generated/custom-css-properties/index.scss' as custom-props;
 
 $grid-columns: 12;
-$grid-gutter-width: awsui.$space-grid-gutter;
 
 .grid {
-  @include styles.styles-reset;
-  display: flex;
-  flex-wrap: wrap;
-  margin: calc(#{$grid-gutter-width} / -2);
-  pointer-events: none;
-
+  // The grid component mostly behaves like a flex layout, but gap with
+  // display flex isn't supported on Safari 13.x and 14.0.
```

Comment:
I use one dimensional grids all the time and often even prefer them to flex so I don't consider this to be hack to be resolved in the future. That being said, as a policy we should consider broader use of @supports particularly for Safari scenarios because our support policy of -3 versions is excessive relative to other vendors.